as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ServerAdaptersTest.java

```diff
@@ -133,7 +133,7 @@ protected File getDownloadPath() {
 	private String getServerHome(String server) {
 		String homeFlag;
 		if (server.contains("WildFly")) {
-			String version = server.split(" ")[1];
+			String version = server.split(" ")[1].replaceAll("\\+","");
```

Comment:
Can you clarify or should this be removed ?

Reply:
This is just quick fix for WildFly 24+ server adapter. I will rewrite it after the WF25 server adapter will be ready.